docs/src/pages/guides/cli-options.md

```diff
@@ -20,13 +20,19 @@ You may want to store different configurations for different environments or pro
 monika --config staging-set.yml
 ```
 
-Configuration files may be placed remotely which you can specify using the same flag and using a URI.
+A neat feature is that the configuration file is watched and any changes will cause Monika to reload.
+
+You can also use a configuration from some remote resource, which you specify using the same `-c` flag and a URI address.
 
 ```bash
 monika -c https://raw.githubusercontent.com/hyperjumptech/monika/main/config_sample/config.desktop.example.yml
 ```
 
-A neat feature is that the configuration file is watched and any changes will cause Monika to reload.
+For remote configuration files, you can additionaly specify how often (in seconds) Monika checks for changes with the `--config-interval` switch like this:
```

Comment:
also mention that the default is 900 seconds.

Reply:
done... mentioned 15 minutes default.